### PR TITLE
2022 February monthly update

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

**Comment by dholbach** [comment]
https://deploy-preview-806--fluxcd.netlify.app/blog/

File: content/en/blog/2022-03-01-february-update/index.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+    OpenShift](/docs/use-cases/openshift/)
+  - [Monitoring with
+    Prometheus](/docs/guides/monitoring/)
+
+Thanks a lot to these folks who contributed to docs and website: Adam
+Dickinson, akirillow, Chanwit Kaewkasi, Davi Garcia, Emanuele Massara,
+Emil Dabrowski, Filipe Sequeira, Hidde Beydals, Ivan Anisimov, Jonathan
+Mourtada, Jørn Fauske, Keith Petersen, Oliver Wiebeck, Patrick
+Cornelißen, Patrick Ruckstuhl, Ricardo Castro, Satyam Kapoor, Somtochi
+Oneykwere, Stacey Potter, Stefan Prodan, Sunny Gogoi and Tamao Nakahara.
```
Comment (kingdonb) [inline content/en/blog/2022-03-01-february-update/index.md:290]:
```suggestion
Onyekwere, Stacey Potter, Stefan Prodan, Sunny Gogoi and Tamao Nakahara.
```

File: content/en/blog/2022-03-01-february-update/index.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+Since the last releases we are also releasing "Software Bill of
+Materials" information, which can be easily parsed programmatically and
+be used in decision making processes about the software in use. Read the
+post to understand how SBOMs are constructed and the different scenarios
+in which they are useful and make you more secure.
+
+[Image Provenance](/blog/2022/02/security-image-provenance/):
+If you have been following news and discussions in the Cloud Native
+space, you will have noticed that this was a hot topic lately. In our
+post we explain why we sign all relevant release artefacts of ours, how
```
Comment (kingdonb) [inline content/en/blog/2022-03-01-february-update/index.md:110]:
```suggestion
post we explain why we sign all relevant release artifacts of ours, how
```

I know this is also a British/American English difference, but I think the Flux project is generally spelling it this way.

**Review by kingdonb — CHANGES_REQUESTED** [review]
LGTM with these updates 👍

**Review by kingdonb — APPROVED** [review]

**Pull request merged**